cli/spark/endpoints/cluster_run.py

```diff
@@ -0,0 +1,18 @@
+import argparse
+import typing
+from cli.spark.aztklib import load_spark_client
+from cli import utils
+
+
+def setup_parser(parser: argparse.ArgumentParser):
+    parser.add_argument('--id',
+                        dest='cluster_id',
+                        required=True,
+                        help='The unique id of your spark cluster')
+    parser.add_argument('command',
+                        help='The command to run on your spark cluster')
+
+def execute(args: typing.NamedTuple):
+    spark_client = load_spark_client()
+    result = spark_client.cluster_run(args.cluster_id, args.command)
+    #TODO: pretty print result
```

Comment:
I think this is worth doing before merging.

Reply:
Not sure how this is best done.

Currently, the output of the command is streaming from each node and printed as it arrives.

There are a couple options to clean this up. First is to wait to print until the entire output of the command is received. Then print the entire output of the command in a single block for each node. This has the advantage of being readable, but provides no feedback to the user until the entire command has completed (will look like it is hanging until then). This could be confusing for commands that provides feedback while they are executing.

The other option is to only print the output of the command from one node. Most of the time, this will be fine. But if there is an error on one node, It's not clear how we would convey that.

Ideally, we could stream the output from each node, and output them in blocks as they come, but I don't think that is possible (at least I'm not sure how it would be done).